packages/model-viewer/src/test/features/scene-graph-spec.ts

```diff
@@ -81,6 +81,32 @@ suite('ModelViewerElementBase with SceneGraphMixin', () => {
     });
   });
 
+  suite('scene export', () => {
+    suite('with a loaded model', () => {
+      setup(async () => {
+        element.src = ASTRONAUT_GLB_PATH;
+
+        await waitForEvent(element, 'load');
+        await rafPasses();
+      });
+
+      test('exports the loaded model to GLTF', async () => {
+        const exported = await element.exportScene( { binary:false } );
+        console.log(exported.size);
+        expect(exported).to.be.not.undefined;
+        expect(exported.size).to.be.equal(3825092);
```

Comment:
This size seems like an implementation detail; I'd trust three to do the right thing even if they change their output json slightly. Maybe just check that it's greater than zero or some reasonable minimum size?

Reply:
We can't test for greater than zero because (at least for the glTF export) the output will always be at least "{}". We can surely test for greater than, let's say, 500 and call it a day but I'm not sure that would be a meaningful test.